include/retdec/fileformat/file_format/file_format.h

```diff
@@ -21,6 +21,20 @@
 namespace retdec {
 namespace fileformat {
 
+/**
+* LoaderErrorInfo - common structure that contains loader error code, error message and user-friendly error message
+*/
+
+struct LoaderErrorInfo
+{
+	LoaderErrorInfo() : loaderErrorCode(0), loaderError(nullptr), loaderErrorUserFriendly(nullptr)
+	{}
+
+	std::uint32_t loaderErrorCode;				// Loader error code, cast to uint32_t
```

Comment:
There are tabs in-between name of the attribute and comment. Please use spaces here. We use tabs only for indentation.

Reply:
Fixed in commit cab945158749486bd6a92ced84237678963a1291